src/extension.ts

```diff
@@ -17,15 +17,18 @@ import { CosmosDBCommands } from './commands';
 import { CosmosDBExplorer } from './explorer';
 import { MongoCommands } from './mongo/commands';
 import { IMongoServer, MongoDatabaseNode, MongoCommand, MongoCollectionNode } from './mongo/nodes';
-import { DocDBDatabaseNode, DocDBCollectionNode } from './docdb/nodes';
-import { CosmosDBResourceNode, INode } from './nodes'
+import { DocDBDatabaseNode, DocDBCollectionNode, DocDBDocumentNode } from './docdb/nodes';
+import { CosmosDBResourceNode, INode } from './nodes';
+import { DocumentClient } from 'documentdb';
 import MongoDBLanguageClient from './mongo/languageClient';
 import { Reporter } from './telemetry';
 
 let connectedDb: MongoDatabaseNode = null;
 let languageClient: MongoDBLanguageClient = null;
 let explorer: CosmosDBExplorer;
 let lastCommand: MongoCommand;
+let lastOpenedDocument: DocDBDocumentNode;
+let lastOpenedDocumentType: string;
```

Comment:
Using a flag here. 
Doesn't look clean, but storing different variables for last Mongo db opened and last DocDB Document opened seemed typesafe to typescript.

Reply:
Use an enum instead. I don't like strings because typing errors can occur more easily